core/block/import/pb/converter.go

```diff
@@ -541,3 +541,7 @@ func (p *Pb) filterObjects(objectTypesToImport widgets.ImportWidgetFlags, object
 	}
 	return rootObjects
 }
+
+func (p *Pb) validateSnapshot(snapshot *pb.SnapshotWithType) bool {
```

Comment:
Maybe rename it to `isSnapshotEmpty`?
1. More specific name for check
2. Starting with `is`, so we understand it returns boolean
3. We do not need to perform negation twice

Reply:
I called it validateSnapshot, because we can add other rules in the future

Reply:
1. Rename to isSnapshotValid 
2. `We do not need to perform negation twice`. What do you mean?

Reply:
Inside isSnapshotValid we perform negation on result of disjunction. It is the first time
Second time we perform it on method's return value (!valid)

Reply:
It is correct, I suppose
If disjunction is positive, it means, that snapshot is not valid. 
If snapshot is empty, the result of function is false - snapshot is not valid. if !valid, then we return error.
Or may be I missed something?

Reply:
Yes, it is correct. Just redundant computing :slightly_smiling_face: 
If validation function will grow and will be reused somewhere else, it is ok to leave it as is

Reply:
I think it's more clear for function, which checks, if snapshot is valid. We can see it from the code, that if snapshot is not valid (!valid), then we return error.